__tests__/app.test.js

```diff
@@ -102,8 +114,9 @@ describe("/api/articles/:article_id", () => {
       return request(app)
         .get("/api/articles/5000")
         .expect(404)
-        .then(({ body }) => {
-          expect(body).toEqual({});
+        .then((res) => {
+          const message = res.body.msg
+          expect(message).toBe("article not found!");
         });
     });
   });
```

Comment:
You are missing a 400 test, what happens if you make a /api/articles/not_an_id ?